src/test/java/com/foo/gosucatcher/domain/estimate/application/ExpertResponseEstimateServiceUnitTest.java

```diff
@@ -0,0 +1,271 @@
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.foo.gosucatcher.domain.estimate.application.dto.request.ExpertResponseEstimateCreateRequest;
+import com.foo.gosucatcher.domain.estimate.application.dto.request.ExpertResponseEstimateUpdateRequest;
+import com.foo.gosucatcher.domain.estimate.application.dto.response.ExpertResponseEstimateResponse;
+import com.foo.gosucatcher.domain.estimate.application.dto.response.ExpertResponseEstimatesResponse;
+import com.foo.gosucatcher.domain.estimate.domain.ExpertResponseEstimate;
+import com.foo.gosucatcher.domain.estimate.domain.ExpertResponseEstimateRepository;
+import com.foo.gosucatcher.domain.estimate.domain.MemberRequestEstimate;
+import com.foo.gosucatcher.domain.estimate.domain.MemberRequestEstimateRepository;
+import com.foo.gosucatcher.domain.expert.domain.Expert;
+import com.foo.gosucatcher.domain.expert.domain.ExpertRepository;
+import com.foo.gosucatcher.domain.item.domain.MainItem;
+import com.foo.gosucatcher.domain.item.domain.SubItem;
+import com.foo.gosucatcher.domain.member.domain.Member;
+import com.foo.gosucatcher.global.error.exception.EntityNotFoundException;
+
+@ExtendWith(MockitoExtension.class)
+class ExpertResponseEstimateServiceUnitTest {
+
+	@InjectMocks
+	private ExpertResponseEstimateService expertResponseEstimateService;
+
+	@Mock
+	private ExpertResponseEstimateRepository expertResponseEstimateRepository;
+
+	@Mock
+	private ExpertRepository expertRepository;
+
+	@Mock
+	private MemberRequestEstimateRepository memberRequestEstimateRepository;
+
+	private Expert expert;
+	private Member member;
+	private MainItem mainItem;
+	private SubItem subItem;
+	private MemberRequestEstimate memberRequestEstimate;
+	private ExpertResponseEstimate expertResponseEstimate;
+
+	@BeforeEach
+	void setUp() {
+		member = Member.builder()
+			.name("이홍섭")
+			.password("q1w2e3")
+			.email("sjun@naver.com")
+			.phoneNumber("010")
+			.build();
+
+		mainItem = MainItem.builder()
+			.name("메인 서비스 이름")
+			.description("메인 서비스 설명").
+			build();
+
+		subItem = SubItem.builder()
+			.mainItem(mainItem)
+			.name("세부 서비스 이름")
+			.description("세부 서비스 설명")
+			.build();
+
+		expert = Expert.builder()
+			.member(member)
+			.storeName("축구 레슨")
+			.location("서울시 강남구")
+			.distance(10)
+			.description("축구 레슨 해드립니다.")
+			.build();
+
+		memberRequestEstimate = MemberRequestEstimate.builder()
+			.member(member)
+			.subItem(subItem)
+			.startDate(LocalDateTime.now().plusDays(1))
+			.detailedDescription("메시가 되고 싶어요")
+			.build();
+
+		expertResponseEstimate = ExpertResponseEstimate.builder()
+			.totalCost(10000)
+			.expert(expert)
+			.memberRequestEstimate(memberRequestEstimate)
+			.description("메시를 만들어 드립니다")
+			.build();
+	}
+
+	@Test
+	@DisplayName("고수 응답 견적서 생성 성공")
+	void createExpertEstimateSuccessTest() throws Exception {
+
+		//given
+		Long expertId = 1L;
```

Comment:
```suggestion
	void createExpertEstimateSuccessTest() throws Exception {
		//given
		Long expertId = 1L;
```
테스트코드 메서드들 전체적으로 개행 없애주셔도 될 것 같습니다!